src/main/java/org/mule/tools/maven/plugin/mule/cloudhub/CloudhubDeployer.java

```diff
@@ -80,8 +80,9 @@ public void deploy() throws DeploymentException
                     String updateMuleVersion = (muleVersion == null) ? app.muleVersion : muleVersion;
                     Integer updateWorkers = (workers == null) ? app.workers : workers;
                     String updateWorkerType = (workerType == null) ? app.workerType : workerType;
+                    Map<String, String> updateProperties = (properties == null || properties.isEmpty()) ? null : properties;
```

Comment:
In the case where properties == null shouldn't you retrieve the existing application properties and use those?